insights/client/apps/ansible/playbook_verifier/__init__.py

```diff
@@ -168,7 +168,7 @@ def exclude_dynamic_elements(snippet):
     """
     if 'insights_signature_exclude' not in snippet.get('vars', {}):
         raise PlaybookVerificationError(
-            "The playbook snippet does not have the key 'vars/insights_signature_exclude', "
+            "Play does not have the key 'vars/insights_signature_exclude', "
```

Comment:
s/Play/The play/ as editor's notes; the same applies to all the changes in this commit, and also affects the tests

Reply:
hm was this done? it seems not

Reply:
It was, it seems GitHub didn't mark it as such though.